recipe/62e7321669f66a90e2a90c8f31af34f59ebc5d7a.patch

```diff
@@ -0,0 +1,71 @@
+From 62e7321669f66a90e2a90c8f31af34f59ebc5d7a Mon Sep 17 00:00:00 2001
+From: Dmitry Arkhipov <grisumbras@gmail.com>
+Date: Thu, 9 Dec 2021 08:11:57 +0300
+Subject: [PATCH] Don't skip install targets if there's <build>no in ureqs
+
+---
+ src/tools/stage.jam      |  4 ++++
+ test/install_build_no.py | 26 ++++++++++++++++++++++++++
+ test/test_all.py         |  1 +
+ 3 files changed, 31 insertions(+)
+ create mode 100755 test/install_build_no.py
+
+diff --git a/src/tools/stage.jam b/src/tools/stage.jam
+index c5f02e3ba4..325129dc81 100644
+--- a/src/tools/stage.jam
++++ b/src/tools/stage.jam
```

Comment:
```suggestion
--- src/tools/stage.jam
+++ src/tools/stage.jam
```

Reply:
Right, I forgot that the file layout of the repo and the tarball used by the recipe are completely different.

Reply:
I am not appying you suggestions because I need to prepend the root path `tools/build/`, will fixup the previous commit instead.

Reply:
Yeah was about to say the suggestion above doesn't take into account that this portion of the source may be located somewhere else. So agree something else is needed